### PR TITLE
[luci] Allow concatenating indices in quantized model

### PR DESCRIPTION
**Pull request by jinevening**
This allows to concatenate indices in quantized model.

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

### CODE REVIEW

File: compiler/luci/pass/src/PropagateQParamBackwardPass.cpp [hunk 1]
```diff
@@ -466,6 +466,9 @@ bool PropagateQParamBackwardPass::run(loco::Graph *g)
     auto circle_node = loco::must_cast<luci::CircleNode *>(node);
     INFO(l) << "PropagateQParamBackwardPass visit node: " << circle_node->name() << std::endl;
 
+    if (circle_node->quantparam() == nullptr)
+      continue;
```
Comment (jinevening) [inline compiler/luci/pass/src/PropagateQParamBackwardPass.cpp:470]: This code is necessary as Bool/S32/S64 type Ops do not have quantparam at this point.

Comment (seanshpark) [reply compiler/luci/pass/src/PropagateQParamBackwardPass.cpp:470]: (optional) maybe add a NOTE or TODO about this?

**Review by seanshpark — APPROVED** [review]
LGTM

**Comment by jinevening** [comment]
The purpose of PR is to quantize the below model (concatenation of int64 tensors).

image

**Review by seanshpark — APPROVED** [review]
LGTM

**Review by mhs4670go — APPROVED** [review]
LGTM

**Pull request merged**